### PR TITLE
Product Rating block: Add the block to the Single Product template by default

### LINKED ISSUE #9287: Product Rating: Add the support for the Single Product Template
Currently, the Product Rating doesn't support the Single Product Template. We have to add it, and we add this block in the default blockified template.

### PR DESCRIPTION
**Pull request by thealexandrelara**
### <samp>🤖 Generated by Copilot at 42381e2</samp>

This pull request adds the `woocommerce/product-rating` block to the block-based single product template. 

### <samp>🤖 Generated by Copilot at 42381e2</samp>
<samp>👨‍💻 Enhanced by @thealexandrelara</samp>

*  Add the `woocommerce/product-rating` block to the `getBlockifiedTemplate` function in `single-product.ts` and the `blockified/single-product.html` file ([link](https://github.com/woocommerce/woocommerce-blocks/pull/9510/files?diff=unified&w=0#diff-84c57bebfa435f9bce1472532eb63de9bacbd17ba965b44cf3dca77fad25e189R37), [link](https://github.com/woocommerce/woocommerce-blocks/pull/9510/files?diff=unified&w=0#diff-7c48ba9797a5f457cf326c28b33e521c8e32a577e82bf67273624dad8fb7220eR55-R56))

Fixes #9287 

#### Accessibility

- [ ] I've tested using only a keyboard (no mouse)
- [ ] I've tested using a screen reader
- [ ] All animations respect [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
- [ ] All text has [at least a 4.5 color contrast with its background](https://webaim.org/resources/contrastchecker/)

#### Other Checks

- [ ] This PR adds/removes a feature flag & I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
- [ ] This PR adds/removes an experimental interfaces and I've updated [this doc](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/blocks/feature-flags-and-experimental-interfaces.md).
- [ ] I tagged two reviewers because this PR makes queries to the database or I think it might have some security impact.

### Screenshots

| Before | After |
| ------ | ----- |
| image | image |

#### Automated Tests
* [ ] Changes in this PR are covered by Automated Tests.
  * [ ] Unit tests
  * [ ] E2E tests

#### User Facing Testing

1. Log in to your WordPress dashboard.
2. From your WordPress dashboard, go to Appearance > Themes. Make sure you have a block-based theme installed and activated. If not, you can install one from the Add New option. Block-based themes include "Twenty-twenty Two," "Twenty-twenty Three", etc.
3. On the left-hand side menu, click on Appearance > Editor. This will open the Site Editor.
4. On the left-hand side menu, click on Templates. This will open the list of available templates.
5. Find and select the Single Product template from the list.
6. When the Classic Product Template renders, click on Transform into Blocks. This will transform the Classic template in a block template if you haven't done it before.
7. Make sure the Product Rating block appears below the Product Title block;
8. On the top right side, click on Save;
9. Visit a product page and make sure the Product Rating block appears below the Product Title.

* [ ] Do not include in the Testing Notes 

### WooCommerce Visibility

* [x] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental

### Changelog

> Add Product Rating block to the default Single Product template

### CODE REVIEW

**Review by albarin — APPROVED** [review]
👏

File: templates/templates/blockified/single-product.html [hunk 1]
```diff
@@ -52,6 +52,8 @@ <h2 class="wp-block-heading">Related products</h2>
 
 			<!-- wp:post-title {"textAlign":"center","level":3,"fontSize":"medium","__woocommerceNamespace":"woocommerce/product-query/product-title"} /-->
 
+			<!-- wp:woocommerce/product-rating {"isDescendentOfSingleProductTemplate":true} /-->
```
Comment (gigitux) [inline templates/templates/blockified/single-product.html]: You added the product rating to the Related Products Block. Should we add the block at line 16?

**Review by gigitux — CHANGES_REQUESTED** [review]
I'm not sure that the `product-rating` block is in the right position in the `.html` template.

File: templates/templates/blockified/single-product.html (hunk 1 above)
Comment (thealexandrelara) [reply templates/templates/blockified/single-product.html]: Ohh I'm sorry, you're correct, it is in the wrong position. I just pushed a new commit fixing that. Thank you for pointing this out

**Comment by mikejolley** [comment]
Bumping to 10.4.0.

**Review by gigitux — APPROVED** [review]
LGTM

**Pull request merged**